### DISCUSSION TITLE
Dual Mining Issues

### DISCUSSION THREAD

**Discussion by thompsontj12** [Lolliedieb/lolMiner-releases#1039] [Q&A]
Anyone else having issues dual mining with 30 series cards? Eth + Ton
My 3060 Ti's don't mine at all on Ton and my 3070's have a very low hashrate

**Comment by mohsenk94** [comment]
Elaborate more please
I am mining fine with:
1x2060S
3x3060TI LHR
4x3070TI LHR
2x3080 LHR
1x3080 TI LHR

Can you explain what is the problem?
Or could you attach a log file?
Hint: For saving log file add this command to the mining batch file:
--log on --logfile /logs/log.txt

**Reply by thompsontj12** [reply]
@mohsenk94 I found out that I needed to start mining with increased power limit to get them to start TON mining. I couldn't get some of them going initially with my regular Eth settings. Do you have any tricks for finding the "sweet spot" with your LHR cards as far as the lhrtune value? Set mine to auto and it continually decreases, never finds the right value. Went to -500 today and was still locking up.

**Reply by donPuerto** [reply]
can you show your bat file settings?

**Reply by thompsontj12** [reply]
@donPuerto 
Rem #################################

set "POOL=us-eth.2miners.com:2020"
set "WALLET=nano_1yoacfxozdy53ycca6hyzgg3i18635jcc63wyqsajjeqsa67aj3iisrpif6b.default"

set "TONPOOL=https://next.ton-pool.com"
set "TONWALLET=EQCuoAPlL5rwIFXL9l_AS1qF6qNZmBXa_Zo21ofGWMu_hGMj"
							
set "EXTRAPARAMETERS=--maxdualimpact 0"

Rem #################################

**Reply by eaamgh** [reply]
Yes, need to raise your PL. Or even remove it. TON is all about Core processing.  Just be careful about your PS. Each one of these cards is now taking about 45W more. On a rig with 6 of them, it means an additional 300W or so... But I've got some pretty interesting results with non-LHR cards. rx6900xt's and a 3070. No impact at all on ETH hashrates.

**Reply by mohsenk94** [reply]
> @mohsenk94 I found out that I needed to start mining with increased power limit to get them to start TON mining. I couldn't get some of them going initially with my regular Eth settings. Do you have any tricks for finding the "sweet spot" with your LHR cards as far as the lhrtune value? Set mine to auto and it continually decreases, never finds the right value. Went to -500 today and was still locking up.

I also leave it at auto, it takes around 30 minutes to tune the entire rig.. Gotta say it doesn't tune the unlocker as well as T-Rex does.. but it's tolerable as long as TON mining is done as well

**Reply by mohsenk94** [reply]
> @donPuerto Rem #################################
> 
> set "POOL=us-eth.2miners.com:2020" set "WALLET=nano_1yoacfxozdy53ycca6hyzgg3i18635jcc63wyqsajjeqsa67aj3iisrpif6b.default"
> 
> set "TONPOOL=https://next.ton-pool.com" set "TONWALLET=EQCuoAPlL5rwIFXL9l_AS1qF6qNZmBXa_Zo21ofGWMu_hGMj"
> 
> set "EXTRAPARAMETERS=--maxdualimpact 0"
> 
> Rem #################################

this:
set "EXTRAPARAMETERS=--maxdualimpact 0"
As I understand in the guide here: https://github.com/Lolliedieb/lolMiner-releases/wiki/Tuning-guide-Dual-TON
It says:
GPU value 0, nearly no TON Mining

Are you intentionally deactivating Ton mining? If not, I suggest you just remove --maxdualimpact switch... Default values are fairly decent imo
